swift_browser_ui_frontend/src/components/Containers.vue

```diff
@@ -56,6 +57,7 @@ export default {
       abortController: null,
       containers: [],
       renderingContainers: [],
+      tableKey: 1,
```

Comment:
What does this do?

Reply:
I noticed that e.g. if there is only one container at the last page, that the table does not update if it is deleted.

Reply:
Meaning the page does not update

Reply:
So if you change that value, the table updates?

Reply:
Sounds like a bug in the data-table, no?

Reply:
Yes, if you change the page, the table updates.

Reply:
Hmm, maybe I have an old version of csc-ui

Reply:
No, update didn't work.

Reply:
I'll remove the key and add an issue to csc-ui